### PR TITLE
Spp 979 new logger

### PR DESCRIPTION
**Pull request by rhodriguerrier**
Add spp logger to ingest takeon and then ensure tests still work by adding environment and survey to payloads.

### CODE REVIEW

File: ingest_takeon_data_method.py [hunk 1]
```diff
@@ -33,36 +35,46 @@ def lambda_handler(event, context):
...
         period = runtime_variables["period"]
         periodicity = runtime_variables["periodicity"]
         previous_period = general_functions.calculate_adjacent_periods(period,
                                                                        periodicity)
         question_labels = runtime_variables["question_labels"]
-        survey_codes = runtime_variables["survey_codes"]
         statuses = runtime_variables["statuses"]
+        survey = runtime_variables["survey"]
+        survey_codes = runtime_variables["survey_codes"]
         input_json = runtime_variables["data"]
```
Comment (Piwington) [inline ingest_takeon_data_method.py:59]: Alphabetical I should be before Q

File: tests/fixtures/test_bricks_wrangler_to_method_runtime.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
     "data": null,
     "bpm_queue_url": "fake_queue_url",
+    "environment": "sandbox",
+    "survey": "BMI_SG",
     "run_id": "bob",
```
Comment (Piwington) [inline tests/fixtures/test_bricks_wrangler_to_method_runtime.json:6]: Alphabetical

File: tests/test_ingest.py [hunk 3]
```diff
@@ -21,6 +21,8 @@
 
 wrangler_runtime_variables_data = {"RuntimeVariables": {
     "bpm_queue_url": "fake_queue_url",
+    "environment": "sandbox",
+    "survey": "BMI_SG",
     "total_steps": "6",
     "run_id": "bob",
```
Comment (Piwington) [inline tests/test_ingest.py:27]: Alphabetical

File: tests/test_ingest.py [hunk 4]
```diff
@@ -53,6 +55,8 @@
 
 wrangler_runtime_variables_bricks = {"RuntimeVariables": {
     "bpm_queue_url": "fake_bpm_queue",
+    "environment": "sandbox",
+    "survey": "BMI_SG",
     "total_steps": "6",
     "run_id": "bob",
     "in_file_name": "test_bricks_method_input",
```
Comment (Piwington) [inline tests/test_ingest.py:62]: Alphabetical

File: tests/test_ingest.py [hunk 5]
```diff
@@ -143,6 +147,8 @@
     "RuntimeVariables": {
         "data": {},
         "bpm_queue_url": "fake_queue_url",
+        "environment": "sandbox",
+        "survey": "BMI_SG",
```
Comment (Piwington) [inline tests/test_ingest.py:151]: Alphabetical

File: tests/test_ingest.py [hunk 6]
```diff
@@ -172,6 +178,8 @@
...
         "run_id": "bob",
         "question_labels": {
```
Comment (Piwington) [inline tests/test_ingest.py:184]: Alphabetical

File: ingest_brick_type_method.py [hunk 7]
```diff
@@ -15,10 +15,12 @@ def handle_error(self, e, data, **kwargs):
         raise ValueError(f"Error validating runtime params: {e}")
 
     bpm_queue_url = fields.Str(required=True)
-    data = fields.List(fields.Dict(required=True))
     brick_questions = fields.Dict(required=True)
     brick_types = fields.List(fields.Int(required=True))
     brick_type_column = fields.Str(required=True)
```
Comment (Piwington) [inline ingest_brick_type_method.py:20]: May seem silly but could I have brick_type_column up 1 as the _ should be above s.

File: ingest_brick_type_method.py [hunk 8]
```diff
@@ -32,28 +34,40 @@ def lambda_handler(event, context):
...
         run_id = event['RuntimeVariables']['run_id']
-
         # Extract runtime variables.
         runtime_variables = RuntimeSchema().load(event["RuntimeVariables"])
-        logger.info("Validated parameters.")
 
         bpm_queue_url = runtime_variables["bpm_queue_url"]
         brick_questions = runtime_variables['brick_questions']
         brick_types = runtime_variables['brick_types']
         brick_type_column = runtime_variables['brick_type_column']
```
Comment (Piwington) [inline ingest_brick_type_method.py:50]: May seem silly but could I have brick_type_column up 1 as the _ should be above s.

**Review by Piwington — CHANGES_REQUESTED** [review]
Alphabetical please.

**Review by Piwington — APPROVED** [review]

**Review by glanvl — APPROVED** [review]

**Pull request merged**